gossipsub.go

```diff
@@ -56,7 +56,9 @@ var (
 	GossipSubGraftFloodThreshold              = 10 * time.Second
 	GossipSubMaxIHaveLength                   = 5000
 	GossipSubMaxIHaveMessages                 = 10
+	GossipSubMaxIDontWantMessages             = 10
```

Comment:
isnt that too low?

Reply:
Yeah, I didn't think enough about this. I just copied it from `GossipSubMaxIHaveMessages`. I think I will change it to 1,000 which will makes the hash map grows by 60,000 per minute (because `GossipSubHeartbeatInterval` is 1 second)